examples/views/dropdown.py

```diff
@@ -0,0 +1,41 @@
+import typing
+
+import discord
+from discord import emoji
+from discord.ext import commands
+
+class Bot(commands.Bot):
+    def __init__(self):
+        super().__init__(command_prefix=commands.when_mentioned_or('$'))
+
+    async def on_ready(self):
+        print(f'Logged in as {self.user} (ID: {self.user.id})')
+        print('------')
+
+# Gives us a view containing a dropdown menu
+class Dropdown(discord.ui.View):
+    def __init__(self, *, timeout: typing.Optional[float] = 180):
+        super().__init__(timeout=timeout)
+
+    # Send a message pinging the user with their selected choice
+    @discord.ui.select(options=[
+        discord.SelectOption(label='Red', description = 'Your favourite color is red', emoji='🟥'), 
+        discord.SelectOption(label='Green', description = 'Your favourite color is green', emoji='🟩'), 
+        discord.SelectOption(label='Blue', description = 'Your favourite color is blue', emoji = '🟦')
+    ], placeholder="Choose your favourite color...")
```

Comment:
1. Set the min and max values manually instead of reying on defaults
2. Subclassing instead of using the decorator interface
3. Added more explanations in comments that are hopefully more descriptive
4. Fixed color vs colour inconsistencies, spaces around '=', and single quotes instead of doubles